views/index.html

```diff
@@ -53,6 +53,9 @@
 	<!-- Angular UI Bootsrtrap -->
 	<script src="angular-ui-bootstrap/dist/ui-bootstrap-tpls.js"></script>
 
+	<!-- Angular Translate -->
+	<script src="angular-translate/dist/angular-translate.min.js"></script>
```

Comment:
Your branch is based on the webpack migration branch, right? If yes, is this script file necessary?

Reply:
I tried it here and worked without the script, thanks!